src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1161,6 +1161,11 @@
 short-summary: Restore a web app from a backup.
 """
 
+helps['webapp config backup delete'] = """
+type: command
+short-summary: Deletes a web app backup.
```

Comment:
Please use the first-person voice in the help message
```suggestion
short-summary: Delete a web app backup.
```